src/microbe_stage/editor/MicrobeEditor.cs

```diff
@@ -965,6 +991,9 @@ private void InitEditor()
 
         UpdateUndoRedoButtons();
 
+        UpdateArrow();
+        editorArrow.Translation = arrowPosition; // Force no lerp on init
```

Comment:
Not really sure but I think the standard convention for Thrive is for comments to be on their own lines(?) @hhyyrylainen

Reply:
Yes. That is the case. I'll add to the long list of styleguide updates that need doing.

Reply:
Should I change those comments as well:
https://github.com/Revolutionary-Games/Thrive/blob/8149a41dd15dfce774b6721fe6a56f1d24dad1e2/src/microbe_stage/editor/MicrobeEditor.cs#L922-L930

Reply:
I'm a bit unsure, but it would fit a bit more with the overall style. On the other hand it makes the LINQ statement even longer.
A bit unrelated, but that LINQ statement is starting to have a bit too much going on for my liking. I still find actually written out code when it gets complicated to be easier to digest and more consistent with other code.